### DISCUSSION TITLE
brew command not found (after single command run on site)

### DISCUSSION THREAD

**Discussion by ar-visions** [Homebrew/discussions#4350] [Everyday usage]
### Output of `brew config`

```shell
command not found
```

### Output of `brew doctor`

```shell
command not found.
```

### Description of issue

you need to add this export PATH=/opt/homebrew/bin:$PATH to the .zshrc

for my case as a user, there is two issues here.

the first is just a kind of unwillingness to touch env configs which seem  a cop-out on installers in general around unix and even windows. it's a legal thing to change a config and software does that.  the text editors that people are forced to run after the fact is doing just that. people love it when things just work so my passion is similar to others on this. for most users, it's never a 'I love it the way I configure a specific shell, so don't touch my configuration'.  its just a different file name on the different shells, and I don't see complaining there from my own experience.  we know the shell env, too.  I would do this across your entire install base as graceful as I could if you would a PR on that.  I don't think I've ever encountered an error writing to user, as user.

second is the perceptibility of its output after install.. I just missed it for years this way, lol.  sorry I guess I shoulda read it but I can overlook quite easy in regards to text on command line.

in my view you can't quite see theres obviously a command to run.  Its syntax starts with ( parens ) which just looks like text with notes at first.  you look past that, or at least I do as I generally never saw the multiples going into a single output.  its a bit sandwiched, just in-between white text lines with ==> blue 5 times (not a unique section when its repeated like so).  It just doesnt stand out, thats all..  and someone like myself misses it constantly. [/kicks-obfuscated-glyph]

**Comment by carlocab** [comment]
This is handled by the install script here: https://github.com/Homebrew/install/blob/010d7b3f54039945a93145d3a5cd91fdd5fbb2bb/install.sh#L1007-L1022

**Comment by ar-visions** [comment]
==> Installation successful!

==> Homebrew has enabled anonymous aggregate formulae and cask analytics.

==> Homebrew is run entirely by unpaid volunteers. Please consider donating:

==> Next steps:

and then this:
- Run these two commands in your terminal to add Homebrew to your PATH:
    (echo; echo 'eval "$(/opt/homebrew/bin/brew shellenv)"') >> /Users/abc/.zprofile
    eval "$(/opt/homebrew/bin/brew shellenv)"
- Run brew help to get started
- Further documentation:
    https://docs.brew.sh


I do welcome this line but I did not ever see it.  the customer should see a framable blob at least.  It should be a different color, like yellow.  this escape sequence: \u001b[33m .. With all these other ==>'s around I just didn't see it all smashed in there.

**Reply by carlocab** [reply]
> The customer should see a framable blob at least. 

Users are not customers. Customers pay money in exchange for a license to use software. We have never taken money in exchange for a license to use Homebrew.

> It should be a different color, like yellow. this escape sequence: \u001b[33m .. With all these other ==>'s around I just didn't see it all smashed in there.

You can open a pull request to improve this at https://github.com/Homebrew/install.

**Reply by carlocab** [reply]
> just so you know not all users have good ability to recognize the text in this fashion.

Sure. That's why I'm asking users who have trouble with it to open a pull request, because they're in the best position to know how to improve it. It's fine to me as is, so I don't think I'm the right person to make this improvement.

**Reply by ar-visions** [reply]
I feel simultaneously blind to this command over so many run attempts and also feel common in case. will issue a PR, as even a 10% reduction in having to google something is a cost savings measure with little effort to put behind it.

**Comment by ar-visions** [comment]
So I am clear on what I missed here:

`(echo; echo 'eval "$(/opt/homebrew/bin/brew shellenv)"') >> /Users/abc/.zprofile eval "$(/opt/homebrew/bin/brew shellenv)"`

Closing because I am duplicating I think and I can see why this is super missed.. I think homebrew could adopt either thing in PR and I'll make one on the simple text formatting and placement.  the other one is more involved but I say doable.
